packages/devtools_app/lib/src/network/network_request_inspector_views.dart

```diff
@@ -420,6 +420,7 @@ class NetworkRequestOverviewView extends StatelessWidget {
     final _colors = [
       searchMatchColor,
       mainRasterColor,
+      mainAsyncColor,
```

Comment:
This is something else I noticed. When we have only two colors available here and we encounter one really short event, there's no visible difference between the events on either side of the effectively invisible event. Having more colors here should help alleviate that particular issue, but I'm not sure if there's any particular reason for only using two in the first place?

Reply:
I figured that's what this is for - I've noticed that issue in the past. Adding a 3rd color sgtm